docs/public/_redirects

```diff
@@ -44,5 +44,17 @@
 /:lang/x/* /x/:splat 301
 # 2023
 /x/whats-new/ https://mui.com/blog/mui-x-v6/ 301
+/x/api/date-pickers/calendar-picker/ /x/api/date-pickers/date-calendar/ 301
+/x/api/date-pickers/calendar-picker-skeleton/ /x/api/date-pickers/day-calendar-skeleton/ 301
+/x/api/date-pickers/month-picker/ /x/api/date-pickers/month-calendar/ 301
+/x/api/date-pickers/year-picker/ /x/api/date-pickers/year-calendar/ 301
+/x/react-data-grid/editing-legacy/ https://https://v5.mui.com/x/react-data-grid/editing-legacy/ 301
+/x/api/date-pickers/desktop-next-date-picker/
+/x/api/date-pickers/desktop-date-picker/ 301
+/x/api/date-pickers/next-date-picker/ /x/api/date-pickers/date-picker/ 301
+/x/react-date-pickers/calendar/
```

Comment:
This lin is strange, are you sure about it? I would expect `in out 301`.

there are a few others as well.

Reply:
Yes, I don't  know why those spaces get replaced by line breaks. Fixed in #8524

Reply:
Perfect, thanks. 

We also have https://app.ahrefs.com/v2-site-explorer/broken-backlinks?anchorRules=&domainNameRules=&followType=all&grouping=one-per-domain&limit=50&mode=prefix&offset=0&refPageTitleRules=&refPageUrlRules=&sort=traffic&sortDirection=desc&surroundingRules=&target=https%3A%2F%2Fmui.com%2F&targetUrlRules= at our disposal to find links to our docs that are broken.